organizations/user.go

```diff
@@ -36,7 +36,7 @@ func GetMember(w http.ResponseWriter, r *http.Request) {
 	orgDoc, _ := utils.GetMongoDbDoc(org_collection, bson.M{"_id": pOrgId})
 	if orgDoc == nil {
 		fmt.Printf("org with id %s doesn't exist!", orgId)
-		utils.GetError(fmt.Errorf("org with id %s doesn't exist", orgId), http.StatusBadRequest, w)
+		utils.GetError(fmt.Errorf("org with id %s doesn't exist!", orgId), http.StatusBadRequest, w)
```

Comment:
The exclamation you added is illegal according to go-checker, it works like eslint in javascript. Syntactically, it's right, but it's not a good practice according to go-checker.